upsertGitHubTag-f79c4f36-3e61-43f4-8f6c-0b2e0e9774d7/deployment/index.js

```diff
@@ -118,23 +144,8 @@ function processEvent(event, callback) {
     // Handle installation events
     var githubEventType = requestBody['X-GitHub-Event']
     if (githubEventType === "installation_repositories") {
-        console.log('Valid installation event');
-        const username = body.sender.login;
-        const installationId = body.installation.id;
-        const repositoriesAdded = body.repositories_added;
-        var repositories = [];
-        repositoriesAdded.forEach((repo) => {
-            repositories.push(repo.full_name);
-        });
-
-        const reposString = repositories.join(",");
-        var pushPostBody = {
-            "installationId": installationId,
-            "username": username,
-            "repositories": reposString
-        };
+        const [pushPostBody,reposString] = handleInstallationRepositoriesEvent(body)
```

Comment:
I guess we don't have a linter, but we typically have a space after a comma.

Reply:
Updated so the function doesn't return multiple values (not to avoid linter issues, but because I believe it makes the code easier to follow).